src/components/flashcard/card-face/card-face.tsx

```diff
@@ -0,0 +1,63 @@
+import './card-face.scss';
+import React from 'react';
+import { ReactComponent as Speaker } from '../../../assets/svg/speaker.svg';
+import { ReactComponent as KebabMenu } from '../../../assets/svg/kebab-menu.svg';
```

Comment:
I think for svgs/icons, we should mark it as ...Icon, so (SpeakerIcon) and (KebabMenuIcon) so we don't mistaken it for a functional component.

Reply:
I branched off this branch. I will update it there and put it in the next PR.